### ISSUE TITLE
Inference low accuracy for Persian Sentence Recognition

### ISSUE THREAD

**Issue by mehrdadazizi72** [PaddlePaddle/PaddleOCR#7815]
I'm training sentence-level recognition for Persian.
The problem is when I'm training the model, the train accuracy is about 70% but when I run infer_rec.py it recognizes just spaces.
My dataset contains 90k synthetic images.

**Train Log:**
[2022/10/03 15:25:41] ppocr INFO: cur metric, acc: 0.8853888883970061, norm_edit_dis: 0.9580471168034234, fps: 3284.1209955926493
[2022/10/03 15:25:41] ppocr INFO: save best model is to ./...../best_accuracy
[2022/10/03 15:25:41] ppocr INFO: best metric, acc: 0.8853888883970061, is_float16: False, start_epoch: 3095, norm_edit_dis: 0.9580471168034234, fps: 3284.1209955926493, best_epoch: 3121
[2022/10/03 15:25:53] ppocr INFO: epoch: [3121/14000], global_step: 4050, lr: 0.001000, acc: 0.727500, norm_edit_dis: 0.813168, loss: 3.354316, avg_reader_cost: 0.00001 s, avg_batch_cost: 0.02504 s, avg_samples: 20.0, ips: 798.73765 samples/s, eta: 28 days, 17:12:59
[2022/10/03 15:25:54] ppocr INFO: save model in ./...../latest
[2022/10/03 15:29:45] ppocr INFO: epoch: [3122/14000], global_step: 4200, lr: 0.001000, acc: 0.735000, norm_edit_dis: 0.809435, loss: 3.171262, avg_reader_cost: 0.11646 s, avg_batch_cost: 0.46399 s, avg_samples: 60.0, ips: 129.31356 samples/s, eta: 28 days, 17:34:35
[2022/10/03 15:29:46] ppocr INFO: save model in ./...../latest

Data Sample and Config File:
[sample.zip](https://github.com/PaddlePaddle/PaddleOCR/files/9703868/sample.zip)
train command:
`python3 -m paddle.distributed.launch --gpus '0,1,2' tools/train.py -c /home/...../PaddleOCR/configs/rec/multi_language/rec_arabic_lite_train_large.yml`

test input:
![fa_1233](https://user-images.githubusercontent.com/38887451/193770283-25caabb3-93dd-43ff-8608-2ee58769e409.png)
test command:
`python3 tools/infer_rec.py -c /home/...../PaddleOCR/configs/rec/multi_language/rec_arabic_lite_train_large.yml  -o Global.pretrained_model=output/rec_arabic_lite_large/best_accuracy │
 Global.infer_img=/home/...../PaddleOCR/train_data/sentence_lvl/test/fa_1233.png `
 [2022/10/04 07:55:12] ppocr INFO: train with paddle 2.3.2 and device Place(gpu:0)                                                                                                               │
W1004 07:55:12.320622 926462 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 7.5, Driver API Version: 11.3, Runtime API Version: 10.2                                               │
W1004 07:55:12.325112 926462 gpu_resources.cc:91] device: 0, cuDNN Version: 8.5.                                                                                                                         │
[2022/10/04 07:55:13] ppocr INFO: resume from ./output/rec_arabic_lite_large/latest                                                                                                                      │
[2022/10/04 07:55:13] ppocr INFO: infer_img: /home/...../PaddleOCR/train_data/sentence_lvl/test/fa_1233.png                                                                                                │
Could not load symbol cublasGetSmCountTarget from libcublas.so.11. Error: /usr/local/cuda-11.3/targets/x86_64-linux/lib/libcublas.so.11: undefined symbol: cublasGetSmCountTarget                        │
[2022/10/04 07:55:14] ppocr INFO:        result:                                0.8420719504356384                                                                                                       │
[2022/10/04 07:55:14] ppocr INFO: success!

**Comment by tink2123** [comment]
Can you get the correct result using the training image prediction? If not, it is recommended to check whether the characters covered in the dictionary are complete. 

In addition, the demo pictures are all long texts. If you use such long data for training and prediction, you need to increase the shape of the training data accordingly, such as [3, 32, 640]

**Comment by mehrdadazizi72** [comment]
@tink2123 Thank you and sorry for my late answer.
I'm retraining my model with new data based on your hints.
For simplicity, I'm producing 120k synthetic training data with a single font, from Persian and English words.
You can see sample data, config, dictionary, and the last few lines of the training log in this file:
[sample.zip](https://github.com/PaddlePaddle/PaddleOCR/files/9960047/sample.zip)
and here is the result for inference a sample form training data:

[2022/11/08 12:50:32] ppocr INFO: profiler_options : None
[2022/11/08 12:50:32] ppocr INFO: train with paddle 2.3.2 and device Place(gpu:0)
W1108 12:50:32.111340 280781 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 11.7, Runtime API Version: 10.2
W1108 12:50:32.114784 280781 gpu_resources.cc:91] device: 0, cuDNN Version: 8.5.
[2022/11/08 12:50:32] ppocr INFO: load pretrain successful from output/rec_arabic_lite_large/best_accuracy
[2022/11/08 12:50:32] ppocr INFO: infer_img: /home/mehrdad/projects/image/PaddleOCR_make_data/data/sentence_lvl/train/fa_0.png
Could not load symbol cublasGetSmCountTarget from libcublas.so.11. Error: /usr/local/cuda-11.2/targets/x86_64-linux/lib/libcublas.so.11: undefined symbol: cublasGetSmCountTarget
#### [1, 179, 115]
[('  ،     ،   ،     ،  ، ', 0.9990041851997375)]
[2022/11/08 12:50:33] ppocr INFO: 	 result:   ،     ،   ،     ،  ، 	0.9990041851997375
[2022/11/08 12:50:33] ppocr INFO: success!

**Comment by mehrdadazizi72** [comment]
@tink2123 Do you have any suggestions for me?

**Comment by ftmasadi** [comment]
Hه, what pre-trained model did you use? Can you send me the download link?

**Comment by mehrdadazizi72** [comment]
@ftmasadi Hi, I did not use any pre-trained model because I didn't find any.
Please provide an email address to send you the download link.

**Comment by ftmasadi** [comment]
Hi, thanks for answering me
I am training a line-level model for sentence recognition. I am at epoch 700. The loss started from high to 300. The loss remains between 70 for several epochs and the accuracy is always 0.0000.

I am using the 30.000 synthetic line segments. I tried the same config with words it's working well but with sentences, it gives unexpected results.
Thank you for your help I am very confused about solving this problem.
my email address:
fatemeh.asadi.z1998@gmail.com

**Comment by mehdiimsv** [comment]
@mehrdadazizi72 
Have you solved this problem? I face the same issue

**Comment by Parisa-Bagherzadeh** [comment]
> [@ftmasadi](https://github.com/ftmasadi) Hi, I did not use any pre-trained model because I didn't find any. Please provide an email address to send you the download link.

سلام

شما قبل از ترین کردن ، روی دیتای فارسی اینفرنس گرفتین؟
اگر آره ،لطفا بگین چجوری؟
